functions/completed-checks/v1.js

```diff
@@ -0,0 +1,256 @@
+'use strict'
+
+const moment = require('moment')
+const sqlService = require('less-tedious')
+const uuid = require('uuid/v4')
+const winston = require('winston')
+const { TYPES } = require('tedious')
+
+winston.level = 'error'
+const config = require('../config')
+sqlService.initialise(config)
+const azureStorageHelper = require('../lib/azure-storage-helper')
+const sqlUtil = require('../lib/sql-helper')
+const markingService = require('./marking.service')
+
+// SQL server vars
+const checkResultTable = '[checkResult]'
+const schema = '[mtc_admin]'
+const checkStatusTable = '[checkStatus]'
+const checkTable = '[check]'
+
+const v1 = {
+  process: async function process (logger) {
```

Comment:
Should also take a message object as param

Reply:
@jon-shipley absolutely, it's something I missed on my first push and later on figured out that it caused all these errors I was getting.